core/src/main/java/com/sequenceiq/cloudbreak/service/stack/ShowTerminatedClusterConfigService.java

```diff
@@ -62,6 +62,13 @@ public ShowTerminatedClustersAfterConfig get() {
         );
     }
 
+    public ShowTerminatedClustersAfterConfig getDefault() {
+        return new ShowTerminatedClustersAfterConfig(
+                defaultShowTerminatedClustersConfig.isActive(),
+                computeShowAfterTimestampMillisecs(defaultShowTerminatedClustersConfig.getTimeout())
+        );
+    }
```

Comment:
After a bit of thinking, this approach does not look so useful either. :smile: `defaultShowTerminatedClustersConfig` was initialized with `showTerminatedOn` (see `init()`), which may be `true` depending on the application deployment settings.

If we were to stick to using the `ShowTerminatedClustersAfterConfig` approach (i.e. not modify `StackDtoService` and `StackDtoRepository`), I would suggest doing this instead:

- Rename the method to `getHideTerminated()`.
- Pass arguments `false, 0L`. The timestamp will be defunct in the query if `showTerminated == false`.
- Actually, since `ShowTerminatedClustersAfterConfig` instances are immutable, the `new ShowTerminatedClustersAfterConfig(false, 0L)` instance could be extracted as a constant and returned for every invocation. (The constant may be kept `private`. Having a dedicated getter instance method like `getHideTerminated()` is nevertheless beneficial for UT: it can be mocked and verified as needed.)